exporter/collector/internal/integrationtest/config/config_test.go

```diff
@@ -12,7 +12,7 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-package integrationtest
+package config
```

Comment:
I had to move this into a separate package because it imports `go.opentelemetry.io/collector/exporter/exporterhelper`, which has an init function creating some instruments/metric producers: https://github.com/open-telemetry/opentelemetry-collector/blob/a8ff2dd98c254538b97348a10c7f270e94991caf/exporter/exporterhelper/obsreport.go#L36-L37. This causes metric descriptors for these to show in the integration tests, but not in the `recordfixtures` script (which doesn't import exporterhelper or run that init).

The fix works because tests in separate packages produce different binaries by default so this stops the side effect for the integration tests.

Reply:
Did you consider importing the exporterhelper package in the record fixtures script?  Or are those metrics we aren't interested in testing?

Reply:
That worked as well to make it pass. Those metrics end up having no points (because no measurements made) but the SD exporter creates metric descriptors. So it's kinda just noise, wdyt?

Reply:
sg